privacy.html

```diff
@@ -153,7 +153,7 @@ <h2>Does this specification introduce new state for an origin that persists acro
       <section>
         <h2>Does this specification expose persistent, cross-origin state to the web?</h2>
         <p>
-Yes. Verifiable Claims contain a random though potentially persistent identifier of the subject. This is passed between the issuer and the inspector. Consequently collusion between them could identify the subject to the inspector, even though the Verifiable Claim itself does not. This is because in many cases the issuer will know the complete identity of the subject, even if the Verifiable Claim only contains a small proportion of it (such as age).
+Yes. Verifiable Claims contain a random though potentially persistent identifier (PId) of the subject. This is passed between the issuer and the inspector. Consequently collusion between them could identify the subject to the inspector, even though the Verifiable Claim itself does not. This is because in many cases the issuer will know the complete identity of the subject, even if the Verifiable Claim only contains a small proportion of it (such as age).
```

Comment:
```suggestion
Yes. Verifiable Credentials contain a random though potentially persistent identifier (PId) of the subject. This is passed between the issuer and the inspector. Consequently, collusion between them could identify the subject to the inspector, even though the Verifiable Credential itself does not. This is because in many cases the issuer will know the complete identity of the subject, even if the Verifiable Credential only contains a small proportion of it (such as age).
```

Reply:
Now in #1288.